x-pack/test/functional/apps/rollup_job/rollup_jobs.js

```diff
@@ -4,64 +4,61 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+
+import datemath from '@elastic/datemath';
 import expect from '@kbn/expect';
-import { indexBy } from 'lodash';
+import { mockIndices } from './hybrid_index_helper';
 
 export default function ({ getService, getPageObjects }) {
+
+  const es = getService('es');
   const esArchiver = getService('esArchiver');
-  const log = getService('log');
-  const PageObjects = getPageObjects(['security', 'rollup', 'common', 'header']);
+  const PageObjects = getPageObjects(['rollup', 'common']);
 
   describe('rollup job', function () {
-    this.tags('smoke');
-    before(async () => {
-      // init data
-      await esArchiver.loadIfNeeded('logstash_functional');
-      await esArchiver.load('canvas/default');
-      await PageObjects.common.navigateToApp('rollupJob');
-    });
+    const rollupJobName = 'rollup-to-be-' + Math.floor(Math.random() * 10000);
```

Comment:
Can we remove `Math.floor(Math.random() * 10000)`?

Reply:
Also, this variable's value is confusingly similar to the value of `indexName`. Can we differentiate them?

Reply:
I think there is some value in leaving the math.floor. If you try to run this test in succession , the fact that the rollup has been created before will not allow you to create it again. So you would have to restart the server between runs which is ridiculous. So I added the random number so the rollup job won't have the same name over and over so you can run the test repeatedly if needbe. I think for the cost,  there is definite value.

Reply:
I think we should definitely leave it.

Reply:
Thanks for the explanation. Could you please leave a comment that explains that we're avoiding naming collisions to support running the test in succession (because ES disallows rollup jobs with the same name, even if the original has been deleted)?

Also, I've found `Date.now()` to be a reliable and terse way to avoid collisions, instead of using `Math.random()`.

Reply:
Fixed.